tensorflow_datasets/scripts/replace_fake_images.py

```diff
@@ -104,10 +104,9 @@ def rewrite_tar(root_dir, tar_filepath):
     # Checking the extension of file to be extract
     if tar_filepath.lower().endswith('gz'):
       extension = ':gz'
-    elif tar_filepath.lower().endswith('bz2'):
+    if tar_filepath.lower().endswith('bz2'):
```

Comment:
Why change this ?

Reply:
I will change it as per previous just forgot to correct it after running locally